core/sdk/unit_tests/clean.sh

```diff
@@ -0,0 +1,28 @@
+
+#! /bin/bash
+
+# Copyright 2019 Intel Corporation
```

Comment:
Update copyright to 2020 here and other files in this PR

Reply:
done